### PR TITLE
fix: style sidebar

### PR DESCRIPTION
**Pull request by bcullman**
### Description
possible way to make the sidebar header stick to the top

### CODE REVIEW

File: .storybook/manager-head.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        position: relative;
+    }
+
+    .sidebar-header > div:first-child {
+        position: relative;
+        margin-right: 0;
+    }
+
+    .sidebar-header > div:last-child {
+        display: none;
```
Comment (jbadan) [inline .storybook/manager-head.html:23]: Per our discussion on slack, I think we need to keep this

Comment (bcullman) [reply .storybook/manager-head.html:23]: 😢

Comment (bcullman) [reply .storybook/manager-head.html]: ok done.

Comment (jbadan) [reply .storybook/manager-head.html]: I know I know it looks so much better with it hidden 😿

**Pull request merged**